lib/task-data/schemas/wait-sel-events.json

```diff
@@ -0,0 +1,37 @@
+{
+    "copyright": "Copyright 2017, Dell EMC, Inc.",
+    "definitions": {
+        "alertFilter": {
+            "type": "object",
+            "properties": {
+                "count": {
```

Comment:
I think other items in alert config can be added here to prevent user's error input as soon as user posting the workflow. Otherwise, the FW update (or other case users would like to use this task) will be executed for hours before the error format is detected.